### DISCUSSION TITLE
remove rate limiting for curators

### DISCUSSION THREAD

**Discussion by michielbdejong** [tosdr/edit.tosdr.org#984] [Ideas] [answered]
if the logged-in user is a curator, we probably shouldn't need to rate limit them

**Comment by madoleary** [comment]
Leaving this here as a potential solution to explore : https://github.com/rack/rack-attack/issues/397

**Comment by michielbdejong** [comment]
Tried changing `req.ip` to `req.ip unless current_user.curator?` in each of the throttle rules in config/initializers/rack_attack.rb but `current_user` is not available there.

**Comment by michielbdejong** [comment]
As a work-around for now I'll temporarily loosen the update-document limit for everyone for now, because that's the one we will be hitting while reviewing old pending points and old doc crawls.

**Comment by JustinBack** [comment, marked as answer]
Added to the roadmap https://git.tosdr.org/tosdr/roadmap/issues/8
